string error_type

string waited_for_topic
mrs_errorgraph_viewer/ErrorgraphNodeID waited_for_node

string ERROR_TYPE_WAITING_FOR_TOPIC = waiting_for_topic
string ERROR_TYPE_WAITING_FOR_NODE = waiting_for_node
string ERROR_TYPE_NOT_REPORTING = not_reporting
string ERROR_TYPE_NO_ERROR = no_error
